tests/www/apply/test_list.py

```diff
@@ -679,6 +682,30 @@ def test_view__filtered_by_selected_job(self):
 
 
 class TestListForSiae:
+    @pytest.mark.parametrize("filter_state", JobApplicationWorkflow.states)
+    def test_message_when_company_got_no_new_nor_processing_nor_postponed_application(self, db, client, filter_state):
+        company = CompanyFactory(with_membership=True)
+        client.force_login(company.members.get())
+        response = client.get(reverse("apply:list_for_siae"), {"states": [filter_state.name]})
+        assertContains(response, "Aucune candidature pour le moment")
+
+    @pytest.mark.parametrize(
+        "state, filter_state",
+        list(itertools.product(JobApplicationWorkflow.PENDING_STATES, JobApplicationWorkflow.states)),
+    )
```

Comment:
Le décorateur est superposable ;) :
```suggestion
    @pytest.mark.parametrize("state", JobApplicationWorkflow.PENDING_STATES)
    @pytest.mark.parametrize("filter_state", JobApplicationWorkflow.states)
```

Reply:
ok!